semconv/README.md

```diff
@@ -16,8 +16,3 @@ Generating semantic convention constants from specification version v1.22.0 at /
 .tools/semconvgen -o semconv/v1.22.0 -t semconv/template.j2 -s v1.22.0 -i /tmp/semantic-conventions/model/. --only=event -p conventionType=event -f generated_event.go
 .tools/semconvgen -o semconv/v1.22.0 -t semconv/template.j2 -s v1.22.0 -i /tmp/semantic-conventions/model/. --only=span -p conventionType=trace -f generated_trace.go
 ```
-
-When generating the constants for a new version ot the specification it is important to note that only
-`generated_trace.go` and `generated_resource.go` are generated automatically. The `schema.go` and `nonstandard.go`
```

Comment:
Looks like we lost `nonstandard.go` at some point, not sure it we need it though

Reply:
I didn't see it in the latest version of the semconv package, so i left it out. If need be we can always add another template 🤷🏻